### DISCUSSION TITLE
Where do I look to know which version of Quarto is bundled with the current RStudio release?

### DISCUSSION THREAD

**Discussion by ghost** [quarto-dev/quarto-cli#4939] [Q&A] [answered]
Where do I look to know which version of Quarto is bundled with the current RStudio release?

**Comment by mcanouil** [comment]
In the HTML header of a rendered document at least 😅
Although this will work only if there is no Quarto other than the one embedded in RStudio.

Is it not an information provided in the info tab where there is RStudio version?
I believe embedded tools available to users should be listed somewhere in the UI with their version.
Probably an issue to open (if it does not exist) on <https://github.com/rstudio/rstudio>.

**Comment by cderv** [comment, marked as answer]
I think the best place is to look into the NEWS file https://docs.posit.co/ide/news/

![image](https://user-images.githubusercontent.com/6791940/227487146-d54ebd5a-4b40-4011-8893-78dd22307715.png)

Otherwise, this can be found in the source code, looking at a tag pinned version of the source code. For example 
https://github.com/rstudio/rstudio/blob/v2023.03.0%2B386/dependencies/common/install-quarto#L30

I have a work item already to discuss with IDE team and other teams at Posit, how we could make this information available as a service 
Discussed in https://github.com/r-lib/actions/issues/469

@allenmanning should we prioritize that maybe ?

**Reply by mcanouil** [reply]
It is kind of standard practice to put those information in "about" tab of software.
For example, VSCode about tab
<img width="274" alt="image" src="https://user-images.githubusercontent.com/8896044/227516302-c03087fd-a7aa-46b0-ac4b-13574734d0a4.png">

**Reply by cderv** [reply]
> It is kind of standard practice to put those information in "about" tab of software.

This would be a feature request for the IDE - I like that. Can you open it if not there already ?

I still need a API way to have the information so that other service like `r-lib/actions`  can know.

**Reply by cderv** [reply]
They have a page with the components though 
![image](https://user-images.githubusercontent.com/6791940/227517399-6e5cfc60-bf2c-459f-bfec-cb101df1801e.png)

But no version in it

**Reply by mcanouil** [reply]
I do not currently have RStudio on any machine, so I wasn't able to check again the about tab.
It appears there are some tools version information at the top, such as Electron (not in a very friendly display though).

I'll see to open an issue later using your screenshot and mine.

**Reply by ghost** [reply]
Thanks folks for the info.  👍
